Firestore/Example/SwiftBuildTest/main.swift

```diff
@@ -304,7 +304,7 @@ func listenToDocuments(matching query: Query) {
 func listenToQueryDiffs(onQuery query: Query) {
   let listener = query.addSnapshotListener { snap, error in
     if let snap = snap {
-      for change in snap.documentChanges {
+      for change in snap.documentChanges() {
```

Comment:
My proposal actually proposed leaving the property, i.e.:

```
let changes = snap.documentChanges
let changesWithMetadata = snap.documentChanges(includeMetadataChanges: true)
```

I don't feel very strongly though and maybe it's better to force a compile-time breaking change. Thoughts?

Reply:
Actually this was a mistake on my part--thinking we couldn't have both a property and a method with the same name (but with a parameter label). Reverting this to a property makes swift happy with both cases so we don't need to break anyone. Much better!